product_stock_state/i18n/fr.po

```diff
@@ -0,0 +1,123 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* product_stock_state
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-10-03 02:23+0000\n"
+"PO-Revision-Date: 2017-10-03 02:23+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: product_stock_state
+#: model:ir.model,name:product_stock_state.model_res_company
+msgid "Companies"
+msgstr "Sociétés"
+
+#. module: product_stock_state
+#: model:ir.model.fields,help:product_stock_state.field_product_category_stock_state_threshold
+msgid "Define custom value under wich the stock state of the products of this category will pass from 'In Stock' to 'In Limited Stock' State. If not set, Odoo will use the value defined for the company"
```

Comment:
wich

Reply:
thanks David, I updated strings and translations